src/ipsis/course-merge/CourseMergeLogDetailCollectionEntity.js

```diff
@@ -33,4 +33,16 @@ export class CourseMergeLogDetailCollectionEntity extends BaseCollectionEntity {
 	loadMorePageSize() {
 		return super.loadMorePageSize(this.getCourseMergeLogs.bind(this));
 	}
+
+	isFiltered() {
+		return this._entity?.properties?.isFiltered;
+	}
+
+	filtersHref() {
+		if (!this._entity.hasLinkByRel(Rels.filters)) {
```

Comment:
Should this be `this._entity?`? We have it here: https://github.com/BrightspaceHypermediaComponents/siren-sdk/blob/1a13fde5f4fcbe03f14ac5a67e4b1d0a39747578/src/ipsis/course-merge/CourseMergeRootEntity.js#L7

Reply:
I don't think it would be necessary as `filtersHref()` wouldn't pinged before loading entity.